buildpipeline/portable-linux.groovy

```diff
@@ -61,11 +61,13 @@ simpleDockerNode('microsoft/dotnet-buildtools-prereqs:rhel7_prereqs_2') {
                                      'Ubuntu.1404.Amd64.Open',
                                      'Ubuntu.1604.Amd64.Open',
                                      'Ubuntu.1610.Amd64.Open',
-                                     // 'Ubuntu.1704.Amd64.Open',
                                      'suse.422.amd64.Open',
                                      'fedora.25.amd64.Open',]
             if (params.OuterLoop) {
-                targetHelixQueues += ['Debian.90.Amd64.Open',]
+                targetHelixQueues += ['Debian.90.Amd64.Open',
```

Comment:
Thanks @weshaggard for the review.

As the are new queues, I plan to put them in outerloop to make sure they work well in CI. In case something is wrong, its impact will be minimum as outerloop runs only take a small portion of all runs.

Once they are proven to be stable, we should do as you suggested, move most important distros into the interloop and everything else in outerloop.